docs/guides/voice/sending-voice.md

```diff
@@ -17,7 +17,7 @@ when developing on .NET Core, this is where you execute `dotnet run`
 from; typically the same directory as your csproj).
 
 For Windows Users, precompiled binaries are available for your 
-convienence [here](https://discord.foxbot.me/binaries/)
+convienence [here](https://discord.foxbot.me/binaries/).
```

Comment:
Dead link, should be removed/replaced with Emzi's link.

Reply:
Agree. Will have to get @foxbot's input first, though.